build-utils/integration-docs-fetch-plugin.js

```diff
@@ -0,0 +1,83 @@
+/*eslint-env node*/
+/*eslint import/no-nodejs-modules:0 */
+const fs = require('fs');
+const https = require('https');
+const path = require('path');
+
+const PLATFORMS_URL = 'https://docs.sentry.io/_platforms/_index.json';
+const DOCS_INDEX_PATH = 'src/sentry/integration-docs/_platforms.json';
+
+const alphaSortFromKey = function(keyExtractor) {
+  return function(a, b) {
+    const nameA = keyExtractor(a);
+    const nameB = keyExtractor(b);
+    if (nameA < nameB) {
```

Comment:
Can you just use `localeCompare` ? `nameA.localeCompare(nameB)`

Reply:
oh, sure, yay!